Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
I suggest put smth like
`#ifdef VCMI_ANDROID` 
`#define VCMI_NO_STD_TOSTRING`
`#endif`
with short comment in macros section of global.h and finish this discussion.